web/src/test/java/org/cbioportal/web/PatientControllerTest.java

```diff
@@ -0,0 +1,243 @@
+    }
+
+    @Test
+    public void getPatientInStudy() throws Exception {
+
+        Patient patient = new Patient();
+        patient.setInternalId(TEST_INTERNAL_ID_1);
+        patient.setStableId(TEST_STABLE_ID_1);
+        patient.setCancerStudyId(TEST_CANCER_STUDY_ID_1);
+        patient.setCancerStudyIdentifier(TEST_CANCER_STUDY_IDENTIFIER_1);
+        CancerStudy cancerStudy = new CancerStudy();
+        cancerStudy.setCancerStudyId(TEST_CANCER_STUDY_ID_1);
+        cancerStudy.setCancerStudyIdentifier(TEST_CANCER_STUDY_IDENTIFIER_1);
+        cancerStudy.setTypeOfCancerId(TEST_TYPE_OF_CANCER_ID_1);
+        cancerStudy.setName(TEST_NAME_1);
+        cancerStudy.setShortName(TEST_SHORT_NAME_1);
+        cancerStudy.setDescription(TEST_DESCRIPTION_1);
+        patient.setCancerStudy(cancerStudy);
+
+        Mockito.when(patientService.getPatientInStudy(Mockito.anyString(), Mockito.anyString())).thenReturn(patient);
+
+        mockMvc.perform(MockMvcRequestBuilders.get("/studies/test_study_id/patients/test_patient_id")
+            .accept(MediaType.APPLICATION_JSON))
+            .andExpect(MockMvcResultMatchers.status().isOk())
+            .andExpect(MockMvcResultMatchers.content().contentType(MediaType.APPLICATION_JSON))
+            .andExpect(MockMvcResultMatchers.jsonPath("$.internalId").doesNotExist())
+            .andExpect(MockMvcResultMatchers.jsonPath("$.stableId").value(TEST_STABLE_ID_1))
+            .andExpect(MockMvcResultMatchers.jsonPath("$.cancerStudyId").value(TEST_CANCER_STUDY_IDENTIFIER_1))
+            .andExpect(MockMvcResultMatchers.jsonPath("$.cancerStudyIdentifier").doesNotExist())
+            .andExpect(MockMvcResultMatchers.jsonPath("$.cancerStudy.cancerStudyId").doesNotExist())
+            .andExpect(MockMvcResultMatchers.jsonPath("$.cancerStudy.cancerStudyIdentifier")
+                .value(TEST_CANCER_STUDY_IDENTIFIER_1))
+            .andExpect(MockMvcResultMatchers.jsonPath("$.cancerStudy.description")
+                .value(TEST_DESCRIPTION_1))
+            .andExpect(MockMvcResultMatchers.jsonPath("$.cancerStudy.name").value(TEST_NAME_1))
+            .andExpect(MockMvcResultMatchers.jsonPath("$.cancerStudy.shortName").value(TEST_SHORT_NAME_1))
+            .andExpect(MockMvcResultMatchers.jsonPath("$.cancerStudy.typeOfCancerId")
+                .value(TEST_TYPE_OF_CANCER_ID_1));
+    }
+
+    @Test
+    public void fetchPatientsDefaultProjection() throws Exception {
+
+        List<Patient> patientList = createExamplePatients();
+
+        Mockito.when(patientService.fetchPatients(Mockito.anyListOf(String.class), Mockito.anyListOf(String.class),
+            Mockito.anyString())).thenReturn(patientList);
+
+        List<PatientIdentifier> patientIdentifiers = new ArrayList<>();
+        PatientIdentifier patientIdentifier1 = new PatientIdentifier();
+        patientIdentifier1.setStudyId(TEST_CANCER_STUDY_IDENTIFIER_1);
```

Comment:
construction of the patientIdentifiers argument (study,patient) pairs.  Construction could also be externalized (previous comment)